bin/fun.js

```diff
@@ -23,7 +23,9 @@ program
   .command('build', 'build the dependencies')
   .command('local', 'run your serverless application locally')
   .command('validate', 'validate a fun template')
-  .command('deploy', 'deploy a fun application');
+  .command('deploy', 'deploy a fun application')
+  .command('init [location]', 'initializes a new fun project');
```

Comment:
```suggestion
  .command('init', 'initialize a new fun project');
```
I have removed options and arguments of subcommands in this overview view to simplify the descriptions. They will show in the detail usage view of commands. Could you please follow this practice?

Reply:
ok